notification/src/handler/notification/notification.handler.js

```diff
@@ -83,23 +108,33 @@ async function updatePaymentWithRepeater(
       )
       break
     } catch (err) {
-      if (err.body.statusCode !== 409)
-        throw new Error(
-          `Unexpected error during updating a payment with ID: ${currentPayment.id}. Exiting. ` +
-            `Error: ${JSON.stringify(serializeError(err))}`
-        )
+      if (err.statusCode !== 409)
+        throw new CommercetoolsError({
+          stack: err.stack,
```

Comment:
I wonder why we try to rewrite the error structure and not just include the error as object with all its properties?

Reply:
we could and capture it, but it has original requests and lots of details, I guess the reason was the sensitive data but it's also useful for tracking. Not sure the way to go here.

Reply:
Why not just:

```
throw new CommercetoolsError({
          error: err,
```
?

Reply:
btw. this library seem promising as it allows to re-throw with original error details: https://www.npmjs.com/package/verror

Reply:
The request is added to the backlog and will be added soon.